build/docker/zap_common.py

```diff
@@ -319,3 +332,47 @@ def zap_get_alerts(zap, baseurl, blacklist, out_of_scope_dict):
         alerts = zap.core.alerts(start=st, count=pg)
     logging.debug('Total number of alerts: ' + str(alert_count))
     return alert_dict
+
+
+def get_latest_zap_client_version():
+    version_info = None
+
+    try:
+        version_info = urlopen('https://pypi.python.org/pypi/python-owasp-zap-v2.4/json')
+    except Exception, e:
```

Comment:
OK I pushed https://github.com/zaproxy/zaproxy/pull/3746/commits/8dec45d27fcb8d0deda7f7cdb89f0f4e7dda4708 and tested on 2.7, 3.6, and 3.4:

```
build/docker - [3356-baseline-check-python-api-version●] » python --version
Python 2.7.13
build/docker - [3356-baseline-check-python-api-version●] » echo 'print(check_zap_client_version())' | python -i ./zap_common.py
>>> None
>>> 

build/docker - [3356-baseline-check-python-api-version●] » python3 --version
Python 3.6.1
build/docker - [3356-baseline-check-python-api-version●] » echo 'print(check_zap_client_version())' | python3 -i ./zap_common.py
>>> None
>>> 

build/docker - [3356-baseline-check-python-api-version●] » python --version
Python 3.4.6
build/docker - [3356-baseline-check-python-api-version●] » echo 'print(check_zap_client_version())' | python -i ./zap_common.py
>>> None
>>> 
```